installing/installing_openstack/installing-openstack-load-balancing.adoc

```diff
@@ -5,4 +5,4 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/installation-osp-balancing-external-loads.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/installation-osp-balancing-external-loads.adoc[leveloffset=+1]
```

Comment:
Where you see the last line in a file being removed and added in this PR, this is the TOC blank line correction script adding new lines to the end of a file where one did not previously exist. It was a side effect of the script, but it is useful.